docs/source/admin/traffic_ops.rst

```diff
@@ -500,7 +500,6 @@ This file deals with the configuration parameters of running Traffic Ops itself.
 	.. versionadded:: 6.2
 
 	:dcdn_id: A string representing this :abbr:`CDN (Content Delivery Network)` to be used in the :abbr:`JWT (JSON Web Token)` and subsequently in :abbr:`CDNi (Content Delivery Network Interconnect)` operations.
-	:jwt_decoding_secret: A string used to decode the :abbr:`JWT (JSON Web Token)` to get information for :abbr:`CDNi (Content Delivery Network Interconnect)` operations.
```

Comment:
Why are we getting rid of this? Or, maybe it makes more sense to ask: if it's unnecessary/unwanted, why was it added in the first place? According to the above `.. versionadded::` directive, this wasn't in 6.1 so we can take it out if we want, no problems there, I'm just curious.

Reply:
i added it originally with the intent to follow the CDNi spec without changing our full system but the more i read into it the more i realized it requires JWT Auth for our full system. so im removing this to just use the `secrets` field above in the cdn.conf like the mojo cookie does

Reply:
Oh, if it's what the `mojolicious` cookie uses then it makes sense to me to use it as the JWT secret.